flask/bin/activate

```diff
@@ -40,7 +40,7 @@ deactivate () {
 # unset irrelevant variables
 deactivate nondestructive
 
-VIRTUAL_ENV="/Users/TexasGuardsman/Desktop/brewtifulworld/flask"
+VIRTUAL_ENV="/Users/TexasGuardsman/Desktop/backenbranch/flask"
```

Comment:
This should probably be set as an environment variable rather than an absolute path on your machine. 1.) Each developer should not have to reset this to run locally. 2.) We should not push personal information to public spaces (for obvious security reasons)